pkg/cephmgr/mon/config.go

```diff
@@ -251,6 +251,7 @@ func ConnectToCluster(context *clusterd.Context, factory client.ConnectionFactor
 		return nil, fmt.Errorf("failed to connect to cluster %s: %+v", cluster.Name, err)
 	}
 
+	logger.Infof("successfully connected to cluster %s", cluster.Name)
```

Comment:
we log the start of the operation with cluster name and user name, if we're going to log the end of the operation it might make sense to log with both those too (add user name to the log statement)